src/stream/src/executor/managed_state/aggregation/extreme.rs

```diff
@@ -395,795 +268,658 @@ where
     }
 
     fn flush(&mut self, _state_table: &mut RowBasedStateTable<S>) -> StreamExecutorResult<()> {
```

Comment:
May leave a "TODO: remove this" here. [#4035](https://github.com/singularity-data/risingwave/issues/4035)